modules/core/src/stat.simd.hpp

```diff
@@ -70,16 +70,14 @@ int normHamming(const uchar* a, int n)
     }
 #endif // CV_POPCNT
 
-#if CV_SIMD128
+#if CV_SIMD
     {
-        v_uint32x4 t = v_setzero_u32();
-        for(; i <= n - v_uint8x16::nlanes; i += v_uint8x16::nlanes)
-        {
-            t += v_popcount(v_load(a + i));
-        }
-        result += v_reduce_sum(t);
+        v_uint64 t = vx_setzero_u64();
+        for(; i <= n - v_uint8::nlanes; i += v_uint8::nlanes)
+            t += v_popcount(v_reinterpret_as_u64(vx_load(a + i)));
```

Comment:
> u64

Does `u32` slower?

Reply:
Yes, u64 could use _mm_sad_epu8 intrinsics that produce u64 results(technically it produce 16-bit results but scatter them as u64)

Reply:
using CV_POPCNT_U64 increase performance when SSE4.2 is enabled